extras/wificfg/wificfg.c

```diff
@@ -1775,24 +1775,60 @@ static void dns_task(void *pvParameters)
     }
 }
 
+/**
+ * @brief      Sanitize SSID. If configuring an AP containing the characters in
+ *             'illegals', the ESP will start an insecure AP named ESP_<macaddr>
+ *             Any illegal characters will be replaced by _
+ *
+ * @param      ssid   the SSID
+ *
+ * @return     true if name got sanitized.
+ */
+static bool sanitize_ssid(char *ssid)
```

Comment:
Just in case the comment on this was lost: this does not seem necessary, these work here.

Reply:
Please see my above ESP_xxxxxx comment. I will revisit this one too.